deploy/crds/noobaa.io_v1alpha1_pvc_cr.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
move this file to internal, it's not a CRD we are creating: deploy/internal/pvc-nsfs.yaml, or something similar.

Reply:
done